core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2448,9 +2538,16 @@ public void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountOptions optio
     Metrics.MOUNT_OPS.inc();
     try (JournalContext journalContext = createJournalContext();
         LockedInodePath inodePath = mInodeTree
-            .lockInodePath(alluxioPath, InodeTree.LockMode.WRITE)) {
-      mPermissionChecker.checkParentPermission(Mode.Bits.WRITE, inodePath);
-      mMountTable.checkUnderWritableMountPoint(alluxioPath);
+            .lockInodePath(alluxioPath, InodeTree.LockMode.WRITE);
+        MasterAuditContext auditContext = createAuditContext("mount", alluxioPath, null,
+            inodePath.getInode())) {
+      try {
+        mPermissionChecker.checkParentPermission(Mode.Bits.WRITE, inodePath);
+        mMountTable.checkUnderWritableMountPoint(alluxioPath);
```

Comment:
This should be outside the `try-catch` since it doesn't deal with client permissions